src/components/search-bar/search-bar.module.css

```diff
@@ -25,6 +25,7 @@
   padding: 15px 20px;
   font-weight: 300;
   font-size: 14px;
+  color: var(--text-primary);
```

Comment:
Please correct the font color, height of the buttons and add an active state to the nav link
![image](https://github.com/vBrusentsov/fe-react-2024/assets/98031768/ca5c1b5c-6c84-4830-bbdd-5890fb1f9760)

Reply:
YOu arrow is lost for an active state 
![image](https://github.com/vBrusentsov/fe-react-2024/assets/98031768/7813fc71-af43-4c9b-a3df-052fc5171df7)

Reply:
Please correct the mobile design 
![image](https://github.com/vBrusentsov/fe-react-2024/assets/98031768/e535b778-f814-49ec-b97c-60c4f541255b)

Reply:
Please correct the layout and add a disable state as per taste description 
![image](https://github.com/vBrusentsov/fe-react-2024/assets/98031768/689d9807-1f92-450b-b69a-e9111ec3179f)